tests/test_generators/test_pydanticgen.py

```diff
@@ -242,6 +243,16 @@ def test_pydantic_enum_descriptions():
         )
 
 
+def test_pydantic_unmasked_keywords(input_path):
+    gen = PydanticGenerator(input_path("unmasked_python_keywords_example.yaml"), package=PACKAGE)
+    code = gen.serialize()
+    mod = compile_python(code, PACKAGE)
+    translation_dict = {"from": "del", "to": "eng"}
+    translation_inst = json_loader.loads(translation_dict, mod.Translation)
+    assert translation_inst.from_ == mod.LanguageEnum("del")
```

Comment:
since the enum template is modified too, we'll want to add a keyword enum option there too and test that it behaves as expected